src/Console.re

```diff
@@ -0,0 +1,126 @@
+type t;
+
+[@bs.new]
+external make:
+  {
+    .
+    "stdout": Stream.t(Stream.writable),
```

Comment:
We want to allow any stream that is a subtype (implements just writable or more)
```suggestion
    "stdout": Stream.t([> Stream.writable ]),
```

Reply:
I'm actually not sure about this. Doing this would allow people to pass a readable (but not writable) stream, and the spec calls for exactly a readable stream. Although I can see maybe a duplex getting passed here... Idk. Thoughts on that?

Reply:
Nevermind, you are right, per our Discord conversation.